### ISSUE TITLE
Add Events category

### ISSUE THREAD

**Issue by luis140219** [jmoenig/Snap--Build-Your-Own-Blocks#709]
The Control category is getting crowded.
I recommend you to add all new Scratch 2.0 blocks, except the ones you've already added.
Add a More Blocks category and change the default colour of new blocks to purple.
Add also an Events category with all When hats, Message reporter and Broadcast blocks

**Comment by studej** [comment]
Hi @luis140219,

I know that devs were talking about adding new categories into Snap!, but it seems that there is no conclusion yet.

1) Some blocks from Scratch 2 you can build on your own (e.g. with `JSfunction` block).
2) We don't need `More blocks` category, because custom blocks always belong into chosen category. 
3) I agree with `Events`. But there is a question: If we split  `Control` into two categories, shouldn't we also split `Variables` category which contains `Variables`, `Lists` and `Other`?. Or should `Variables` and `Lists` be in one category (as in Scratch under `Data`)? Is `Other` as separated category usefull?  Devs must decide :-)

**Comment by jmoenig** [comment]
Yeah, I've seen the other categories in Scratch 2.0, but I'm not enthusiastic about adding more categories at the moment
